### DISCUSSION TITLE
[Particle] Add action to change particle based on json

### DISCUSSION THREAD

**Discussion by Keevle** [4ian/GDevelop#2345] [Enhancement ideas (and their technical discussions)]
Right now there isnt any way to live preview particles,so the only way to work with them is to painstakinly open up the edit menu,change a variable then preview it just to see a change.
I recently found [Pixi Particle Editor](https://pixijs.io/pixi-particles-editor/#gas),It would be great if Gdevelop could read the exported json data and change the currrent particle objects settings based on that data via a action like
`Particle Fire: Load particle data 'fire.json'`

**Comment by 4ian** [comment]
Should be covered by https://trello.com/c/Ce46QxKj/417-add-support-for-hot-reload-when-a-preview-is-running-and-play-button-is-clicked-continue-running-the-game-with-updated-events-ob :)

**Comment by Keevle** [comment]
It would still be great if we could Load/Extract particle settings and save them in some sort of string
